contents/blog/mykonos-hackathon.md

```diff
@@ -0,0 +1,172 @@
+
+Then we took on our data processing agreement, or DPA, to create a generator that makes this hugely exciting task even more fun. You can quickly populate your own form, select the data region, and, if you want, add some pizazz with fairy tale or Taylor Swift mode. DPA? [Try DPYAY!](/dpa)
+
+### Managed Reverse Proxy
+
+<BuiltBy people={["Frank Hamand"]} />
+
+Everyone loves ad-blockers. But, for a lot of our customers, they stop data from reaching PostHog. You can already deploy a reverse proxy to PostHog Cloud to get around this, but it’s a somewhat convoluted process that requires you to jump through 16 hoops and login to AWS. [Our docs on this are great](/docs/advanced/proxy), but Frank decided to build a better solution.
+
+During the hackathon, he built the reverse proxy functionality right into PostHog. The option is tucked away in the PostHog settings. Simply add in any domain you control and the system will spit out a CNAME that you then need to set in your DNS provider. Wait a few seconds for the update to happen and voila, the reverse proxy is live.
+
+### HERMES
+
+<BuiltBy people={["Annika Schmid", "Simon Fisher", "Mine Kansu"]} />
+
+At PostHog, we love speaking to our users. Maybe a bit too much. Right now, our master customer interviews doc is 382 pages long and contains almost 200 user interviews. It’s a great resource, but it’s getting a bit unwieldy. But now we’ve entered a bold new era of feedback management at PostHog thanks to HERMES, or, snappily, Holistic Evaluation Repository for Managing Enhancements and Suggestions.
+
+This is effectively a database of user interviews, showing who was interviewed, who they work for, what they do, how much they pay for PostHog, the products they talked to us about, and an AI-generated summary of our user interview notes. The database is searchable and you can easily add new interviews in a couple of clicks.
+
+The database is linked right into PostHog, making it easy to see associated user and organization profiles. It’s also hooked into Vitaly, our customer success tool, to automatically pull in more customer and business information.
+
+HERMES also uses ChatGPT to generate a summary of the features requested during the interview based on the human-authored interview notes. This makes it easy to share actionable feedback from users directly with the product team responsible for that feature. 
+
+As part of the project, we also revamped our system for categorizing and tracking feature requests from customers, making it easier for us to see what people want and prioritize the most important product work.
+
+### Data crunching
+
+<BuiltBy people={["Sandy Spicer", "Tim Glaser"]} />
+
+![Data crunching](https://res.cloudinary.com/dmukukwp6/image/upload/hackathon_datacrunch_525d9e362b.png)
```

Comment:
This screenshot doesn't have the actual stats on it.

Reply:
i'm not sure any of them do, actually? i've run a bunch of queries and it always just says 'crunching through hogloads of data.' will update the copy so it's accurate based on what's live.
<img width="1275" alt="Screenshot 2024-05-30 at 2 11 53 PM" src="https://github.com/PostHog/posthog.com/assets/162187331/b6c043a5-293c-44ca-8c04-d2d503110ad3">
ching through

Reply:
Hmm, let me have a look....

Reply:
![Screenshot 2024-05-31 at 15 04 32](https://github.com/PostHog/posthog.com/assets/92976667/b76442ef-8a04-4d50-998c-fcc47a2cb0e9)

It just requires a more chunky one to make it work.

Reply:
ah ha! thanks, adding this in